### DISCUSSION TITLE
Using same wallet in app and webapge

### DISCUSSION THREAD

**Discussion by sebastian-baier** [Web3Auth/Web3Auth#1147] [Help]
Hello,
I have two applications
- a webpage (nextjs)
- an app (react native)

Now I want my users to log into both with web3auth and let them use the same wallet.
I use the same client ID in both applications but I don't get the same wallets although I use the same provider.
Is this currently even possible or am I doing something wrong with configuration?

Help would be apreciated

**Comment by guru-web3** [comment]
Web3Auth accounts are scoped to applications (clientId). All those demos must be using different clientId and hence you are getting different accounts for same login methods.

You can get your application clientId by registering your project at https://dashboard.web3auth.io/.

Other reasons can be:-

Different verifiers being used
Different client ids
Different social logins (email passwordless and google is not same)
Torus wallet accounts are different from web3auth accounts.

**Reply by sebastian-baier** [reply]
the problem was the network for me
just works with mainnet no other network is working

**Comment by sebastian-baier** [comment]
in my case to make it work, is switching with both applications to mainnet instead of other networks
That's the only way I got it to work
Don't know if this is expected behavior
